src/components/common/author/Author.vue

```diff
@@ -1,19 +1,19 @@
 <template>
     <div>
```

Comment:
```suggestion
    <div>
```
it does not make sense to have 2 div wrappers, one is enough